packages/storybook/context/analytics.context.js

```diff
@@ -0,0 +1,80 @@
+import Vue from 'vue';
+
+// Note this was upto date at the time of publishing
+// but does not need to be aligned/maintained as only
+// a placeholder.
+const platformData = {
+    environment: 'storybook',
+    name: 'storybook',
+    appType: 'web',
+    applicationId: 7,
+    userAgent: undefined,
+    branding: undefined,
+    country: undefined,
+    language: undefined,
+    jeUserPercentage: undefined,
+    currency: undefined,
+    version: undefined,
+    instancePosition: undefined,
+    isPilot: undefined
+};
+
+const userData = {
+    'a-UserId': 'storybook',
+    authType: undefined,
+    email: undefined,
+    globalUserId: undefined,
+    signinType: undefined,
+    signupDate: undefined
+};
+
+const pageData = {
+    group: 'storybook',
+    name: undefined,
+    httpStatusCode: 200,
+    conversationId: undefined,
+    requestId: undefined,
+    orientation: undefined,
+    display: undefined
+};
+
+// Just placeholder for Storybook to use in place of the global `$gtm`
+// It merely console logs what it can under its limited capacity
+export default () => {
+    const gtm = {
+        pushPlatformData: ({
+            featureName, locale, customFields
+        } = {}) => {
+            let data = { ...platformData, name: featureName, language: locale };
+            if (customFields) {
+                data = { ...data, ...customFields };
+            }
+            console.log('pushPlatformData', { platformData: { ...data } });
```

Comment:
Do we need these console.logs?

Reply:
Yes so you at least check in storybook that some form of output is being produced from you code (albeit not actually a GTM event)